2018-03-17T18:27:03.371Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: error executing chaincode: transaction returned with failure: [composerchannel-1906948e]Calling chaincode Invoke() returned error response [Error: Object with ID 'singo@carge-network' in collection with ID 'Participant:org.gov.fundtracker.MMDA' does not exist]. Sending ERROR message back to peer
    at /home/vagrant/hyperledger/server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-03-17T18:27:03.373Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: error executing chaincode: transaction returned with failure: [composerchannel-1906948e]Calling chaincode Invoke() returned error response [Error: Object with ID 'singo@carge-network' in collection with ID 'Participant:org.gov.fundtracker.MMDA' does not exist]. Sending ERROR message back to peer","stack":"Error: Error trying to query business network. Error: error executing chaincode: transaction returned with failure: [composerchannel-1906948e]Calling chaincode Invoke() returned error response [Error: Object with ID 'singo@carge-network' in collection with ID 'Participant:org.gov.fundtracker.MMDA' does not exist]. Sending ERROR message back to peer\n    at channel.queryByChaincode.then.catch (/home/vagrant/hyperledger/server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:820:34)\n    at <anonymous>"}$
2018-03-17T18:27:03.587Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: error executing chaincode: transaction returned with failure: [composerchannel-760e8ed8]Calling chaincode Invoke() returned error response [Error: Object with ID 'singo@carge-network' in collection with ID 'Participant:org.gov.fundtracker.MMDA' does not exist]. Sending ERROR message back to peer
    at /home/vagrant/hyperledger/server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-03-17T18:27:03.587Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: error executing chaincode: transaction returned with failure: [composerchannel-760e8ed8]Calling chaincode Invoke() returned error response [Error: Object with ID 'singo@carge-network' in collection with ID 'Participant:org.gov.fundtracker.MMDA' does not exist]. Sending ERROR message back to peer","stack":"Error: Error trying to query business network. Error: error executing chaincode: transaction returned with failure: [composerchannel-760e8ed8]Calling chaincode Invoke() returned error response [Error: Object with ID 'singo@carge-network' in collection with ID 'Participant:org.gov.fundtracker.MMDA' does not exist]. Sending ERROR message back to peer\n    at channel.queryByChaincode.then.catch (/home/vagrant/hyperledger/server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:820:34)\n    at <anonymous>"}$
